Assets/MixedRealityToolkit/Attributes/MixedRealityControllerAttribute.cs

```diff
@@ -6,6 +6,10 @@
 using System;
 using System.Linq;
 
+#if WINDOWS_UWP && !ENABLE_IL2CPP
```

Comment:
did we get any comments in ship room whether we would support .net? From what i understand newer versions of unity don't provide .net tooling?

Reply:
The standing policy is to support the backends that Unity supports. 2018.4 will continue to support the .NET backend for 2 years following its release.

As we move to Unity 2019, we will only support il2cpp. For as long as we support Unity 2018, we will also support .NET.

Reply:
Could this be simplified to just `#if ENABLE_DOTNET`?

Reply:
I wasn't aware of that define! We should create an issue to replace all of the  #if WINDOWS_UWP && !ENABLE_IL2CPP instances with it!